src/addrdb.h

```diff
@@ -73,4 +73,15 @@ class CBanDB
     bool Read(banmap_t& banSet);
 };
 
+/** Dump the anchor IP address database (anchors.dat) */
+void DumpAnchors(const fs::path& anchors_db_path, const std::vector<CAddress>& anchors);
+
+/**
+ * Reads the anchor IP address database (anchors.dat)
+ *
+ * Anchors are last known outgoing block-relay-only peers that are
+ * tried to re-connect to on startup.
+ */
+std::vector<CAddress> ReadAnchors(const fs::path& anchors_db_path, bool remove_after_read = true);
```

Comment:
What is the `remove_after_read` param for? It's not used in the one place this is called, so should be removed.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).